test/basic.js

```diff
@@ -81,6 +87,14 @@ assert.strictEqual(string.get.hwb('hwb(240, 100%, 50.5%)1234'), null);
 
 // generators
 assert.equal(string.to.hex([255, 10, 35]), '#FF0A23');
+assert.equal(string.to.hex([255, 10, 35, 1]), '#FF0A23');
+assert.equal(string.to.hex([255, 10, 35], 1), '#FF0A23');
+assert.equal(string.to.hex([255, 10, 35, 1]), '#FF0A23');
+assert.equal(string.to.hex([255, 10, 35], 1), '#FF0A23');
+assert.equal(string.to.hex([255, 10, 35, 0.3]), '#FF0A234D');
+assert.equal(string.to.hex([255, 10, 35], 0.3), '#FF0A234D');
+assert.equal(string.to.hex([255, 10, 35, 0.3]), '#FF0A234D');
+assert.equal(string.to.hex([255, 10, 35], 0.3), '#FF0A234D');
```

Comment:
Yes, sorry. I had noticed a few duplicates in a few other spots, so wasn't sure if it was an intentional practice or not. I definitely take those out.